ncbi-tools-bin/6.1.20170106-6-deb/Dockerfile

```
@@ -1,20 +1,20 @@
-FROM biocontainers/biocontainers:vdebian-buster-backports_cv1
+FROM biocontainers/biocontainers:debian-buster-backports_cv2
 MAINTAINER biocontainers <biodocker@gmail.com>
 LABEL    software="ncbi-tools-bin" \ 
-    base_image="biocontainers/biocontainers:vdebian-buster-backports_cv1" \ 
+    base_image="biocontainers/biocontainers:debian-buster-backports_cv2" \ 
     container="ncbi-tools-bin" \ 
     about.summary="NCBI libraries for biology applications (text-based utilities)" \ 
     about.home="http://www.ncbi.nlm.nih.gov/IEB/ToolBox/" \ 
-    software.version="6.1.20170106-6-deb" \ 
+    software.version="6.1.20170106+dfsg1-0+deb10u2" \ 
```

Let's revert to '6.1.20170106-6-deb' in the file, to avoid creating a separate folder.
```suggestion
    software.version="6.1.20170106-6-deb" \ 
```